lib/clusterManager.js

```diff
@@ -523,6 +604,93 @@ if (cluster.isMaster) {
     );
   }
 
+  ClusterManager.prototype.getStorageByKey = function(pluginId, key) {
+    return this.callClusterMethodRemote(null, "clusterManager", "getStorageByKey", [pluginId, key],
+    function(resultHandler) { 
+      return resultHandler[0]; // Return the first arg (storage object) of getStorage
+    },
+    function(e) {
+      clusterLogger.warn("ZWED0168W - Unable to retrieve storage value from cluster", e);
+    }
+    );
+  }
+
+  ClusterManager.prototype.getStorageAll = function(id) {
+    return this.callClusterMethodRemote(null, "clusterManager", "getStorageAll", [id],
+      function(resultHandler) {
+        return resultHandler[0]; // Return the first arg (storage object) of getStorageAll
+      },
+      function(e) {
+        clusterLogger.warn("ZWED0164W - Unable to retrieve storage object from cluster", e);
+        return e;
+      }, 5000 /* We need to give ample timeout time since this method is used to verify 
```

Comment:
This needs to be configurable by server.json, something in the "node" object, but maybe this needs to be in a section specific to clustering, such as node.cluster.storageTimout=5000